src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -367,7 +370,7 @@ case object CODESIZE extends ConstOp(0x38)(_.env.program.length)
 case object CODECOPY extends OpCode(0x39, 3, 0, _.G_verylow) {
   protected def exec[W <: WorldStateProxy[W, S], S <: Storage[S]](state: ProgramState[W, S]): ProgramState[W, S] = {
     val (Seq(memOffset, codeOffset, size), stack1) = state.stack.pop(3)
-    val bytes = state.program.getBytes(codeOffset.toInt, size.toInt)
+    val bytes = OpCode.sliceBytes(state.program.code, codeOffset, size)
```

Comment:
Yes, it's used for PUSHx (the semantics would be the same, but no unnecessary UInt256 -> Int conversions)